src/varname.jl

```diff
@@ -297,10 +298,18 @@ subsumes(t::PropertyLens, u::PropertyLens) = false
 # FIXME: Does not support `DynamicIndexLens`.
 # FIXME: Does not correctly handle cases such as `subsumes(x, x[:])`
 #        (but neither did old implementation).
-subsumes(t::IndexLens, u::IndexLens) = _issubindex(t.indices, u.indices)
-subsumes(t::ComposedLens{<:IndexLens}, u::ComposedLens{<:IndexLens}) = subsumes_index(t, u)
-subsumes(t::IndexLens, u::ComposedLens{<:IndexLens}) = subsumes_index(t, u)
-subsumes(t::ComposedLens{<:IndexLens}, u::IndexLens) = subsumes_index(t, u)
+subsumes(
+    t::Union{IndexLens,ComposedLens{<:IndexLens}},
+    u::Union{IndexLens,ComposedLens{<:IndexLens}}
+) = subsumes_index(t, u)
+
+
+const subsumedby = (t, u) -> subsumes(u, t)
```

Comment:
The same holds for the rest of the `const`, right? As in, we prefer `⊒(args...) = subsumes(args...)`, etc. too?

Reply:
No, in the other cases (apart from the last one) we want the Unicode character to be an alias such that we can use the ASCII and Unicode names interchangeably. Otherwise we define a new function that just forwards all arguments to another function but dispatches, ChainRules definitions etc. of one of these functions wouldn't cover the other one.